game/views.py

```
@@ -588,42 +586,110 @@ def register_view(request):
+            # Fetch latest committed database state
+            existing_user_by_username = User.objects.filter(username__iexact=username).first()
+            existing_user_by_email = User.objects.filter(email__iexact=email).first()
+
+            if existing_user_by_username and existing_user_by_email and existing_user_by_username == existing_user_by_email:
+                if not existing_user_by_username.is_active:
+                    # Case B: Exact match inactive account. Trigger secure re-verification.
+                    user_to_verify = existing_user_by_username
+                else:
+                    conflict_type = 'exact_active'
+            else:
+                if existing_user_by_email:
```

Ruff (0.15.15)

[warning] Use `elif` instead of `else` then `if`, to reduce indentation

Convert to `elif`

(PLR5501)